check_hp_msa.py

```diff
@@ -385,6 +388,10 @@ def parse_args():
     args.add_argument('--insecure', help='Do not check certificates', action='store_true')
 
     args.add_argument('--version', '-V', help='Print version', action='store_true')
+    args.add_argument('--authentication_hash_algo',
```

Comment:
Can we make this something shorter? `authentication-hash` or `auth-hash-type`

Also, dashes in the Flag are translated by the argpase to `_`

> Any internal - characters will be converted to _ characters

Reply:
better?